lib/routes/webtoons/naver.js

```diff
@@ -0,0 +1,39 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const domain = 'https://comic.naver.com';
+
+async function getDescription(link) {
+    const { body } = await got.get(link);
```

Comment:
should use cache for invariable content here, see: https://docs.rsshub.app/en/joinus/#enable-caching

Reply:
I fixed it. please check it.